src/App.vue

```diff
@@ -158,13 +158,17 @@ export default class App extends Mixins(DateMixin) {
   }
 
   async created (): Promise<void> {
-    // update Current Js Date now and every 1 minute thereafter
-    await this.updateCurrentJsDate()
-    this.updateCurrentJsDateId = window.setInterval(this.updateCurrentJsDate, 60000)
+    // start with "page not ready" - show spinner
+    // (spinner will be hidden in landing.vue)
+    this.showSpinner = true
 
     // listen for spinner show/hide events
     this.$root.$on('showSpinner', (flag = false) => { this.showSpinner = flag })
```

Comment:
Note: The event handler has to be in place before the async call on line 169, otherwise landing.vue gets mounted and sends the event to hide it before the handler exists!